bazel/api-gen/rendering/index.ts

```diff
@@ -15,12 +17,24 @@ interface EntryCollection {
 function parseEntryData(srcs: string[]): EntryCollection[] {
   return srcs.map((jsonDataFilePath) => {
     const fileContent = readFileSync(jsonDataFilePath, {encoding: 'utf8'});
-    return JSON.parse(fileContent) as EntryCollection;
+    const fileContentJson = JSON.parse(fileContent) as unknown;
```

Comment:
IS the unknown cast here necessary vs casting straight to `EntryCollection`?

Reply:
`unknown` is technically correct since we don't know what it is.

I could have it be `EntryCollection | DocEntry` if you would prefer